### PR TITLE
Fixed : Deprecated Warinings

### LINKED ISSUE #3169: Deprecated PHP warnings
### Describe the bug

Extension of https://github.com/openemr/openemr/issues/3165#issuecomment-599254869
 
Fix PHP deprecated warnings

### Expected behavior

NO more deprecated warnings

### PR DESCRIPTION
**Pull request by yashrajbothra**
Signed-off-by: Yash Bothra <yashrajbothra786@gmail.com>

Fixes #3169

### CODE REVIEW

**Comment by yashrajbothra** [comment]
I this allowed to make a direct call or I did it very wrong?

**Comment by bradymiller** [comment]
hi @yashrajbothra ,
It does solve this issue :) Since I am not a huge Zend fan, I initially leaned towards going this way. But I think better to instead make the underlying `public function escapeLimit` function static in /interface/modules/zend_modules/module/Application/src/Application/Plugin/CommonPlugin.php

**Comment by yashrajbothra** [comment]
Definitely i can go that way but there are others also with similar situation. I will try to make them static aswell.

File: interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php [hunk 1]
```diff
@@ -595,8 +596,8 @@ public function getAllergies($pid, $encounter)
                 }
 
                 if ($row['reaction']) {
-                    $reaction_text = \Carecoordination\Model\CarecoordinationTable::getListTitle($row['reaction'], 'reaction', '');
-                    $reaction_code = \Carecoordination\Model\CarecoordinationTable::getCodes($row['reaction'], 'reaction');
+                    $reaction_text = getListTitle($row['reaction'], 'reaction', '');
+                    $reaction_code = getCodes($row['reaction'], 'reaction');
```
Comment (bradymiller) [inline interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php]: these don't look right. Should they look like:
```
$reaction_text = CarecoordinationTable::getListTitle($row['reaction'], 'reaction', '');
$reaction_code = CarecoordinationTable::getCodes($row['reaction'], 'reaction');
```

Comment (yashrajbothra) [reply interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php]: Thanks for the review @bradymiller 😄

**Comment by bradymiller** [comment]
hi @adunsulag , I just wanted to makes sure we weren't reverting any changes you made in the zend module regarding the static issues (I remember there being some complicated reasons regarding static php warnings and wanted to make sure we weren't going in the wrong direction on this PR).

**Comment by adunsulag** [comment]
@bradymiller This shouldn't revert any changes and I skimmed through the changed files and didn't see any problems.  We'd have to search the codebase to know for sure, or run the CCDA import/export.

**Comment by bradymiller** [comment]
@adunsulag , Thanks for checking this out. Testing well, so bringing this into codebase.
@yashrajbothra , Thanks for the fixes! -brady

**Pull request merged**